assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -39,7 +39,7 @@ const FieldRenderer: React.FC<FieldRendererProps> = (props) => {
 				{label}
 				{info && (
 					<Tooltip placement='right' tooltip={info}>
-						<InfoCircleOutlined className='ee-form-item__help-info' />
+						<InfoCircleOutlined />
```

Comment:
![image](https://user-images.githubusercontent.com/1477453/82234950-c32ed280-993a-11ea-9bf9-486ef2899704.png)

Reply:
love it 👍 great job